assets/locales/en.po

```diff
@@ -763,6 +763,23 @@ msgstr ""
 "As a last resort, you can reset your password in the “forgot your password” section of your Cozy. "
 "If you did not initiate this request, contact us by replying directly to this email."
 
+msgid "Mail Update Email Subject"
+msgstr "Confirm your email"
+
+msgid "Mail Update Email Intro 1"
+msgstr "Hello %s,"
+
+msgid "Mail Update Email Intro 2"
+msgstr ""
+"You asked to change your password for Cozy Pass. "
```

Comment:
I don't get the link with Cozy Pass.

And aren't we updating the e-mail address rather than the passphrase?

```suggestion
"You asked to change your email address. "
```

Reply:
I have taken the Password confirmation as example. I didn't see the part.